WRT-280: Fix info how to add features after release (#43)
tradingview/rest_integrations_docs: source/Integration_overview.rst

Reviewer: > The sandbox is a fully functional copy of the TradingView website located at `beta-rest.xstaging.tv`_. Access to the
> resource is provided by adding an IP address to the whitelist on the TradingView side.

Addition to your changes. Can we clarify this paragraph a bit (L135)? Who adds an IP address to the whitelist? What should broker do to get access?

Author: I've clarified this paragraph within another task https://jira.xtools.tv/browse/WRT-445. I will pass it to you soon c:

* Turn off HSTS in production.
* Wait for two years.

Adding features after release
...............................

If you want to add new features to the TradingView production environment after release,
you first need to confirm the changes with the TradingView team.

> Once confirmed, you can implement the new feature on your :ref:`staging environment <trading-environments>`.

Reviewer 2: Этот шаг обязательный. Тогда может быть не 'can' а 'should', или как-то перефразировать. Но тут надо у english-proofreading спрашивать.
Смысл в том, что без проверки на их стеджинге, как это сказано в 192 строке.

> Note that deploying new features to your production environment is only allowed after successful testing by the TradingView team.

Reviewer: This sentence is a bit too long and overwhelmed with gerund and passive voice. Let's try to make it simpler. 
```suggestion
Note that new features can be deployed to your production environment only after the TradingView team successfully test them.
```
or
```suggestion
Note that the TradingView team will deploy new features to your production environment only after successful testing.
```
